tests/index.php

```diff
@@ -13,7 +13,7 @@
 
 // Set the Guzzle client options
 $options = [
-  'base_uri' => MaibClient::MAIB_TEST_BASE_URI,
+  'base_url' => MaibClient::MAIB_TEST_BASE_URI,
```

Comment:
Constant is already `URI` already, why changing key name to `url`? https://www.hostinger.com/tutorials/uri-vs-url.
Every url is uri, but not every uri is url.